src/Makefile.am

```diff
@@ -2741,12 +2741,10 @@ tests_testEventLoop_LDFLAGS = $(LIBADD_DL)
 ## Run the unit tests. check_PROGRAMS only builds the binaries.
 TESTS += $(check_PROGRAMS) testHeaders
 
-## Special Universal .h dependency test script
-## aborts if error encountered
-testHeaders: $(srcdir)/*.h $(srcdir)/DiskIO/*.h $(srcdir)/DiskIO/*/*.h
-	$(SHELL) $(top_srcdir)/test-suite/testheaders.sh "$(CXXCOMPILE)" $^ || exit 1
+# Can't use SOURCES because we need to filter out unit tests
```

Comment:
Why do we need to filter out unit tests? We exclude unit test headers from the same fairly universal "builds standalone" guarantee we are trying to uphold for other headers?

Reply:
It looks like the latest code is using SOURCES, so this PR-added Makefile.am comment is stale and should be removed. I am not attaching a suggestion here because this whole custom testHeaders blob (including the old comment right after it) should be removed (and src/TestHeaders.am included) instead AFAICT.

Reply:
yes. Removing